desktop_version/src/Script.cpp

```diff
@@ -2857,6 +2857,10 @@ void scriptclass::startgamemode( int t )
         map.resetplayer();
         map.gotoroom(game.saverx, game.savery);
         map.initmapdata();
+
+        cl.generatecustomminimap();
+        map.customshowmm = true;
+
```

Comment:
`map.customshowmm` is reset to `true` in `scriptclass::hardreset` anyways, so this instance is unnecessary (and all the other instances of `map.customshowmm = true;` should probably have been removed in the first place).

The bigger problem with this, though, is that you're doing this in case 20, which is only when the level editor is entered. At that point, a level has not yet been loaded, so the generated minimap will always be completely blank. The `cl.generatecustomminimap();` should be moved to case 21, which is when in-editor playtesting is entered.